x-pack/plugins/cases/server/client/cases/push.ts

```diff
@@ -69,18 +63,13 @@ export const push = async (
     actionsClient,
     user,
     logger,
+    auditLogger,
+    authorization,
   } = clientArgs;
 
-  /* Start of push to external service */
-  let theCase: CaseResponse;
-  let connector: ActionResult;
-  let userActions: CaseUserActionsResponse;
-  let alerts;
-  let connectorMappings;
-  let externalServiceIncident;
-
   try {
-    [theCase, connector, userActions] = await Promise.all([
+    /* Start of push to external service */
```

Comment:
I wrapped the whole function in a single try/catch. We don't need the extra errors for each section because the calls being made also log/throw a descriptive error.